contracts/interactors/AaveInteractor.sol

```diff
@@ -69,7 +69,7 @@ abstract contract AaveInteractor is Initializable {
     address aToken = IPool(aavePool).getReserveData(token).aTokenAddress;
 
     uint256 totalBalanceScaledBefore = IAToken(aToken).scaledBalanceOf(address(this));
-    IERC20(token).approve(aavePool, amount);
+    IERC20(token).forceApprove(aavePool, amount);
```

Comment:
I don't think we need this change for two reasons:

- Aave pool _should_  (verification needed) take all the approved tokens, so basically your allowance is always zero after the `stake` call
- As an alternative making an extra `approve(aavePool, 0)` call in the end of `_aaveStake` is much better because you are not leaving non-zero allowance 'outside' of tx
- Increased gas costs

I'd either ignore that completely (my pick) or add zero-amount approve rather than using `forceApprove`